src/EditorFeatures/CSharpTest/ExtractMethod/SelectionValidatorTests.cs

```diff
@@ -1520,7 +1520,7 @@ public async Task AnonymousTypeMember1()
     {
         var code = """
             using System;
-            class C { void M() { {|r:var x = new { {|b:String|} = true }; |}} }
+            class C { void M() { {|r:var x = new { {|b:String|} = true };|} } }
```

Comment:
note: the test previously were non-functional.  they were not actually validating that the spans unless a special bit was set on a reseult avlue (which wasn't computed properly).  I also remove this bit in a follow up pr because the whole "feature adds special out of band data, for tests to validate, but which doesn't impactthe feature itself" is an awful paradigm.